### DISCUSSION TITLE
FormButtonGroup.FormItem 显示”:“ 符号

### DISCUSSION THREAD

**Discussion by iscmcloud** [alibaba/formily#1626] [Q&A 2.x] [answered]
版本：2.0.0-beta.69
问题：FormButtonGroup.FormItem 显示”:“ 符号，而 2.0.0-beta.5*不会显示。此元素用来显示按钮组，不需要显示“:” 符号在label

**Comment by janryWang** [comment, marked as answer]
开发分支已经修复
